### PR TITLE
quick review on Sergey's Abramov tasks

### PR DESCRIPTION
**Pull request by brouhaha88**
Trouble with naming

- [x] @sergiy17

### CODE REVIEW

File: abramov_module.rb [hunk 1]
```diff
@@ -57,28 +57,37 @@ def abr_182(x)
   a = []
   for_v = []
```
Comment (brouhaha88) [inline abramov_module.rb]: please explain full name

File: abramov_module.rb [hunk 2]
```diff
@@ -57,28 +57,37 @@ def abr_182(x)
   a = []
```
Comment (brouhaha88) [inline abramov_module.rb]: please explain full name

File: abramov_module.rb [hunk 3]
```diff
@@ -57,28 +57,37 @@ def abr_182(x)
...
   sum_of_dig << for_v.length
   sum = 0
+
   for_v.each { |s| sum += s }
+
   puts sum_of_dig.to_s
 end
 
 ###
 def count_divs(n)
```
Comment (brouhaha88) [inline abramov_module.rb]: please explain full name

File: abramov_module.rb [hunk 4]
```diff
@@ -89,8 +98,11 @@ def g_all(x)
 
 def abramov_323(n)
   dvs_of_n = g_all(n)
```
Comment (brouhaha88) [inline abramov_module.rb]: please explain full name

File: abramov_module.rb [hunk 5]
```diff
@@ -57,28 +57,37 @@ def abr_182(x)
...
 def count_divs(n)
   div_sum = 0
+
   (1...n).each { |d| div_sum += d if n % d == 0 }
+
   div_sum
 end
 
 def abramov_560(a, b)
   friendly_pairs = []
```
Comment (brouhaha88) [inline abramov_module.rb]: please explain  name

File: abramov_module.rb (hunk 4 above)
Comment (brouhaha88) [inline abramov_module.rb]: please explain full name

**Comment by brouhaha88** [comment]
- You need to add some spaces since this code is not yet uglified.
- Naming is impossible to understand.
- Please comment on the naming above so i can fix this.

File: abramov_module.rb (hunk 2 above)
Comment (brouhaha88) [inline abramov_module.rb]: a

File: abramov_module.rb (hunk 5 above)
Comment (sergiy17) [inline abramov_module.rb]: div_sum it's OK.without this **abramov_560** will not wok

File: abramov_module.rb (hunk 1 above)
Comment (sergiy17) [inline abramov_module.rb]: i saw some mistakes so i changed small part of  **abramov_182**

**Pull request closed without merging**